gm4_liquid_minecarts/data/liquid_minecarts/functions/init_tank.mcfunction

```diff
@@ -0,0 +1,22 @@
+#@s = liquid minecart
+#run from try_to_unload
+
+data modify block ~ ~ ~ CustomName set from entity @s CustomName
+summon armor_stand ~ ~-.45 ~ {CustomName:"\"gm4_liquid_tank_display\"",Tags:["gm4_no_edit","gm4_liquid_tank_display","gm4_lm_needs_texture"],NoGravity:1,Marker:1,Invisible:1,Invulnerable:1,Small:1,DisabledSlots:2039552,Fire:20000}
+data modify entity @e[type=armor_stand,tag=gm4_lm_needs_texture,limit=1,sort=nearest] ArmorItems[3] set from entity @e[type=armor_stand,tag=gm4_liquid_minecart_display,limit=1,sort=nearest] HandItems[0]
+tag @e[type=armor_stand] remove gm4_lm_needs_texture

```

Comment:
Why is there a tag for wether the tag needs to be applied if it's removed right after? If its not needed, it would save on an `@e`